### DISCUSSION TITLE
Welcome to ergpy Discussions!

### DISCUSSION THREAD

**Discussion by mgpai22** [mgpai22/ergpy#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Grav1006** [comment]
Hello! 
I'm trying to create a token with ergpy , but when I run the code, I get an error ("Address has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0" )
But I installed JDK17 from Oralce site.
How can I solve this problem?

**Reply by mgpai22** [reply]
Can you try with Java 11?

**Reply by Grav1006** [reply]
Thanks, this helped! 
But now, when I run the code, I get an error:
'SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder'
'SLF4J: Defaulting to no-operation (NOP) logger implementation'
How can i fix this?

**Reply by mgpai22** [reply]
This should be a default error it has to do with the dependencies of the jar, everything comes out of. Does the code still run?

**Reply by Grav1006** [reply]
Thank you very much! Despite the error, the code is executed and the coin comes to the wallet
